pipeline_dp/budget_accounting.py

```diff
@@ -202,7 +214,9 @@ def __init__(self,
     def request_budget(self,
                        noise_kind: NoiseKind,
                        sensitivity: float = 1,
-                       weight: float = 1) -> MechanismSpec:
+                       weight: float = 1,
+                       count: int = 1,
```

Comment:
Please raise NotImplementedError exception when count != 1 and noise_standard_deviation != None

Reply:
Done